app/frontend/src/locales/ja/translation.json

```
@@ -61,7 +61,9 @@
         "showSupportingContent": "サポート内容の表示",
         "speakAnswer": "音声による回答",
         "info": "情報",
-        "save": "保存"
+        "save": "保存",
+        "copy": "回答をコピー",
```

```suggestion
        "copy": "コピーする",
```
@bnodir Would this be correct Japanese for just "Copy"? We're thinking of moving from "Copy answer" to "Copy".

---

@pamelafox We can simplify "回答をコピー" to just "コピー". For consistency, since we are omitting "する" in "保存" (save), it makes sense to do the same with "コピー". For example, in the OS context menu, "コピー" is commonly used.

---

```suggestion
        "copy": "コピー",
```